### PR TITLE
[AB#40224] service ID and urls are renamed from encoding to video

### PR DESCRIPTION
**Pull request by AxTrusov**
# Pull Request

## Prerequisites

- [x] My code follows the coding conventions
- [x] All tests pass

## Description

Urls and service ID reference updated to use `video` instead of `encoding`.

### CODE REVIEW

File: services/entitlement/service/README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ The `yarn setup:webhooks` script is used for development environments. To get
 the values for `ENTITLEMENT_WEBHOOK_SECRET` and `MANIFEST_WEBHOOK_SECRET` env
 variables for the production environment, the following mutation must be
 launched against the Video Service GraphQL API. You can use GraphiQL UI for that
-(https://encoding.service.eu.axinom.net/graphiql):
+(https://video.service.eu.axinom.net/graphiql):
```
Comment (AxTrusov) [inline services/entitlement/service/README.md:86]: This URL is currently not accessible. Need to make sure it is before this PR is completed

File: services/media/service/src/generated/messaging/rascal-schema.json [hunk 2]
```diff
@@ -407,7 +407,7 @@
         },
         "media-service:cue_point_types:declared": {
```
Comment (AxTrusov) [inline services/media/service/src/generated/messaging/rascal-schema.json]: Currently, a change like this would make sure that we receive messages using the updated service ID, but:
1. the previously existed queue` ax-encoding-service:cue_point_types:declare` in media-template-specific vhost will continue to exist.
2. The queue for events, e.g. `media-service:cue_point_types:declared` would have bindings for both service IDs: 
image
   a. This will result in media service receiving a message 2 times because of our backward compatibility logic.

In short - we need to perform some sort of RabbitMQ cleanup to remove obsolete queues and bindings

File: services/entitlement/service/README.md (hunk 1 above)
Comment (Zehelein) [reply services/entitlement/service/README.md:86]: Wrote with Anushka and he said the change is in the repo and once it is pushed the URL will work.

File: services/media/service/src/generated/messaging/rascal-schema.json (hunk 2 above)
Comment (Zehelein) [reply services/media/service/src/generated/messaging/rascal-schema.json]: Yes right - the cleanup should be done also as part of this PR - thanks.

Comment (Zehelein) [reply services/media/service/src/generated/messaging/rascal-schema.json]: Was done as part of https://github.com/Axinom/mosaic-media-template/pull/140/files

**Review by Zehelein — APPROVED** [review]
